resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
remember I am not talking about the math behind it. because then:
```
curl http://localhost:8080/users?offset=10&limit=10
curl http://localhost:8080/users?page=2&limit=10
```
would be the same of course. I am talking about the intentions a client has by describing it in  the query params.